docs/source/models.rst

```diff
@@ -2654,6 +2654,77 @@ avoid failed inference requests.
 
 \***** If using `pyfunc` in MLflow Model Serving for realtime inference, the raw audio in bytes format must be base64 encoded prior to submitting to the endpoint. String inputs will be interpreted as uri locations.
 
+Using inference_config and dictionary parameters for `transformers` inference
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+For `transformers` inference, there are two ways to pass in additional arguments to the pipeline.
+
+* Use inference_config when saving/logging the model
+* Use dictionary parameters during `predict` function call
```

Comment:
@serena-ruan do we have a follow-up to support these arguments for `mlflow.pyfunc.spark_udf()`?

cc @BenWilson2

Reply:
Updated in this PR